pkg/gateway/client/apiactivity.go

```diff
@@ -41,7 +41,7 @@ func (c *Client) ActiveUsersByDate(ctx context.Context, start, end time.Time) ([
 			Pluck("user_id", &ids).Error; err != nil {
 			return err
 		}
-		return tx.Where("id IN (?) AND NOT internal", ids).Find(&users).Error
+		return tx.Where("id IN (?) AND NOT internal AND deleted_at IS NULL", ids).Find(&users).Error
```

Comment:
Do we want to include deleted users here? I am excluding them for now, but we can change it back in order to include them.

Reply:
I think this is correct.